ext/mysql2/mysql2_ext.h

```diff
@@ -9,7 +9,13 @@ void Init_mysql2(void);
 #define RSTRING_NOT_MODIFIED
 #include <ruby.h>
 
-#ifdef HAVE_MYSQL_H
+#ifdef HAVE_SERVER_MYSQL_H
+#include <mysql.h>
+#include <server/mysql_com.h>
+#include <errmsg.h>
+#include <mysqld_error.h>
+#include <server/mysql_version.h>
```

Comment:
Looks like https://github.com/MariaDB/server/commit/3ec96c18244b448b2adf30deb856e34907c12326#diff-46193e2e675adcde81f1d095d8b3e1db is the culprit?

> MDEV-13370 Ambiguous behaviour regarding installation of header files
> install all server includes under /usr/include/mysql/server/ and C/C includes under /usr/include/mysql/